app/src/main/java/com/example/helloworkflow/HelloWorkflow.kt

```diff
@@ -18,7 +18,7 @@ object HelloWorkflow : StatefulWorkflow<Unit, State, Nothing, Rendering>() {
 
     data class Rendering(
         val message: String,
-        val onClick: (Unit) -> Unit
+        val onClick: () -> Unit
```

Comment:
Don't you have to edit the call site too?

Reply:
Probably, I just did this from my phone so I wouldn't forget